flowey/flowey_hvlite/src/pipelines/copilot_setup_generator.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use anyhow::Result;
+
+/// Generate copilot-setup-steps.yml for GitHub Copilot Coding Agent
+#[derive(clap::Args)]
+pub struct CopilotSetupGeneratorCli {}
+
+impl CopilotSetupGeneratorCli {
+    pub fn run(self, repo_root: &std::path::Path) -> Result<()> {
+        let output_file = repo_root.join("copilot-setup-steps.yml");
+
+        let content = generate_copilot_setup_steps();
+
+        std::fs::write(&output_file, content)?;
+
+        println!("Generated copilot-setup-steps.yml");
+        Ok(())
+    }
+}
+
+fn generate_copilot_setup_steps() -> String {
+    format!(
```

Comment:
Use a template `yml` file that sits in the repo, like `gh_flowey_bootstrap_template.yml`